### ISSUE TITLE
ASR with Beaver-dam-7B

### ISSUE THREAD

**Issue by zui-jiang** [AoiDragon/HADES#4]
What's the threshold did you use when evaluating VLM's harmness with Beaver-dam-7B?

**Comment by zui-jiang** [comment]
btw when using LLaVA-1.5-13B (greedy generate), nearly all the response to google-images is like "I'm sorry...., It's not ethical....", which I think is a well defines behaviors for harmless instructions. So is it possible that the high ASR is caused by Beaver-dam-7B?

**Comment by zui-jiang** [comment]
btw when using llava-1.5-13B to get response to the google toxic image alongsides the instructions, most of the response is like "I'm sorry...  It is not ethnical....", which I think is a well-defined response for harmless instructions, and when using llama-guard 2 as a judger for the generated response, I got a ASR 6% for Animal category which is much slower than the beaver-dam-7B's results. 

Could you offer the original model's response and a detail judger parameter setting for reproducement?

**Comment by AoiDragon** [comment]
Hello @zui-jiang ,

For the first question, we followed the default settings of Beaver-dam-7B by setting the threshold to 0.5.

For the second question, we manually checked the generation results of evaluated models and ensured that most of the judging results of Beaver-dam-7B are reliable. The discrepancies you noticed may be due to differences in generation config. We used the default generation parameters for LLaVA-1.5 (as specified in model_vqa.py), with `temperature = 0.2` and `top_p = None`.
